.github/workflows/pull_request.yml

```diff
@@ -29,6 +29,28 @@ jobs:
           java-version: 'adopt:1.11.0-11'
           node-version: '16.7.0'
 
+      - name: Cache sbt dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/*.sbt') }}
+          restore-keys: |
+            ${{ runner.os }}-sbt-
+
+      - name: Cache sbt build artifacts
+        uses: actions/cache@v2
+        with:
+          path: |
```

Comment:
These could be helpful but they don't really reduce the build time by much (~2m), I have an internal version which I'll push

Reply:
Thank you for your reply, Alex. I appreciate your feedback on the pull request and it's great to hear that you have an internal version that can further improve the build time. In that case, would you like me to close this pull request for now? Please let me know your thoughts